manifests/crd/acceleratorprofile.opendatahub.io.crd.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Well, after approving it... I realize, do you want only to have the spec or do you want the operator to apply it?
If so, you need to add it into the `kustomize` file, I'll wait for your confirmation @Gkrumbach07 if that's the case

Reply:
operator is going to need to apply it

Reply:
Great, so last changes are fine now!